CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone
+# opens a pull request.
+*       @blootsvoets @nivemaham @fnobilia
```

Comment:
Good point, especially for new features. I'm a bit worried that the review process will slow progress for small changes like documentation updates or urgent bug fixes.